samcli/commands/deploy/command.py

```diff
@@ -28,14 +31,14 @@
 """
 
 
-@configuration_option(provider=TomlProvider(section="parameters"))
 @click.command(
     "deploy",
     short_help=SHORT_HELP,
     context_settings={"ignore_unknown_options": False, "allow_interspersed_args": True, "allow_extra_args": True},
     help=HELP_TEXT,
 )
-@template_click_option(include_build=False)
+@configuration_option(provider=TomlProvider(section="parameters"))
```

Comment:
is the order of decorators important?

Reply:
not after, we made template `is_eager=True`

Reply:
okay